packages/payload-auth/src/better-auth/plugin/lib/build-collections/users/endpoints/signup.ts

```diff
@@ -62,7 +62,7 @@ export const getSignupEndpoint = (pluginOptions: BetterAuthPluginOptions): Endpo
             'Content-Type': 'application/json'
           },
           body: JSON.stringify({
-            name: '',
+            name: 'New User',
```

Comment:
We should not set "New User" and instead add the name field instead in our forms.